src/Modal/GlobalModal.tsx

```diff
@@ -22,6 +23,21 @@ export function hideGlobalModal(key: string) {
   DeviceEventEmitter.emit(HIDE_GLOBAL_MODAL, key)
 }
 
+const layoutAnimation = new Layout().delay(CHILD_ANIM_DURATION)
+  .duration(LAYOUT_ANIM_DURATION)
+  .build()
+
+// Using duration of 1ms to disable the animation(sort of)
+const disabledLayoutAnimation = new Layout().duration(1).build()
```

Comment:
A duration of -1 should finx the issue discrussed in #1

Reply:
Awesome, I have added that. I didn't get a chance to test this. Didn't know we had to pass -1 to disable the animation, good finding on that.